frontend/pulse3d/src/components/table/UploadsSubTable.js

```diff
@@ -4,33 +4,52 @@ import Checkbox from "@mui/material/Checkbox";
 
 const Container = styled.div`
   padding: 0 3.5rem;
+  background-color: #ececed8f;
 `;
 
 const SubContainer = styled.div`
+  width: 100%;
   display: flex;
-  justify-content: space-between;
   align-items: center;
-  margin: 0 1rem;
 `;
 const SubHeader = styled.div`
-  display: flex;
-  justify-content: space-between;
   background-color: var(--dark-blue);
   color: white;
-  padding: 0.4rem 2rem;
+  padding: 0.4rem 0;
   font-size: 0.85rem;
   border-radius: 3px;
+  display: flex;
 `;
-const SubRow = styled.div`
-  font-size: 0.75rem;
-  margin: 1rem 0;
+
+const FilenameHeader = styled.div`
+  padding-left: 3.9%;
+  width: 40%;
+`;
+const DateHeader = styled.div`
```

Comment:
Couldn't you reuse a header styled div  at 20% instead of having three of the same?